src/js/plugins/plugin.resume.js

```diff
@@ -40,14 +40,29 @@ BookReader.prototype.getResumeValue = function() {
 }
 
 /**
- * Set's the page resume value, for remembering reader's page
+ * Return cookie path using pathname up to /page/... or /mode/...
+ * using window.location.pathname for urlPathPart:
+ * - matches encoding
+ * - ignores querystring part
+ * - ignores fragment part (after #)
+ * @param {string} urlPathPart - window.location.pathname
+ */
+BookReader.prototype.getCookiePath = function(urlPathPart) {
+  return urlPathPart.match('.+?(?=/page/|/mode/|$)')[0];
```

Comment:
Thanks for the tests!
Q: is `page|mode` the only parts of the URL that immediately proceed the item identifier?

Reply:
Yes, for now. '/search/ used to be there but no longer.